### PR TITLE
Upgrade to python 3.8

### PR DESCRIPTION
**Pull request by symroe**

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -10,22 +10,24 @@ pytest-cov = ">=2.6,<3.0"
...
 django-apiblueprint-view = ">=2.3.0,<3.0"
 raven = ">=6.10,<7.0"
 django-cors-headers = ">=2.0.0,<3.3.0"
 aws-wsgi = {ref = "unquote_path_strings", git = "https://github.com/DemocracyClub/awsgi.git"}
-dc-base-theme = { ref = "0.3.12", git = "https://github.com/DemocracyClub/dc_base_theme.git" }
+dc-base-theme = { ref = "0.3.13", git = "https://github.com/DemocracyClub/dc_base_theme.git" }
 whitenoise = "*"
 sentry-sdk = "*"
 watchtower = "*"
+aws-sam-cli = "==1.37.0"
```
Comment (GeoWill) [inline Pipfile:27]: Duplicate of line 13

File: aggregator/settings/base.py [hunk 2]
```diff
@@ -103,15 +104,15 @@
...
 STATIC_URL = "/static/"
 
-from dc_theme.settings import (  # noqa
-    get_pipeline_settings,
-    STATICFILES_STORAGE,
-    STATICFILES_FINDERS,
-    SASS_INCLUDE_PATHS,
-)
+from dc_utils.settings.pipeline import *  # noqa
+from dc_utils.settings.pipeline import get_pipeline_settings
```
Comment (GeoWill) [inline aggregator/settings/base.py:108]: I would expect 108 is redundant after 107

**Review by GeoWill — COMMENTED** [review]
This looks good. 
One general point/question is that a lot of dependencies in the Pipfile are pinned to specific versions. Will this not stop us getting minor version updates from dependabot? So if they're currently `foo==n` can't they be  `foo<n+1`. If they do need to be pinned hard then I wonder if it's worth noting why for future yak shavers?

File: aggregator/settings/base.py (hunk 2 above)
Comment (symroe) [reply aggregator/settings/base.py:106]: FYI I removed this, but then flakes gave an error that I couldn't figure out how to get around. Added back in, I assume like I did last time I did this!

**Pull request merged**